disnake/ui/select.py

```diff
@@ -323,8 +343,10 @@ def select(
     max_values: :class:`int`
         The maximum number of items that must be chosen for this select menu.
         Defaults to 1 and must be between 1 and 25.
-    options: List[:class:`disnake.SelectOption`]
-        A list of options that can be selected in this select menu.
+    options: Union[List[:class:`disnake.SelectOption`], List[:class:`str`], Dict[:class:`str`, :class:`str`]]
+        A list of options that can be selected in this menu. Use explicit :class:`SelectOption`s
```

Comment:
```suggestion
        A list of options that can be selected in this menu. Use explicit :class:`SelectOption`\s
```
(again same thing above)

Reply:
Done ^^